sched/wqueue/kwork_thread.c

```diff
@@ -161,7 +170,15 @@ static int work_thread(int argc, FAR char *argv[])
            */
 
           leave_critical_section(flags);
+
+#ifdef CONFIG_WQUEUE_TAG_WORKER_NAME
+          sinfo("WORKER %s started\n", name_tag);
+#endif
           CALL_WORKER(worker, arg);
```

Comment:
Is it enough to print worker address? we can get the function name from address with any methods(e.g. System.map, addr2line, gdb).

Reply:
Printing worker name is useful if the full debug log is analyzed what was happening, to analyze how all workers alternate to each other (the whole picture). If just worker addresses printed, it would be necessary to translate each address occurrences using System.map or addr2line, thus the full log not easily readable.